spec/graphql/types/int_spec.rb

```diff
@@ -26,21 +26,77 @@
         assert_equal(-(2**31), GraphQL::Types::Int.coerce_result(-(2**31), context))
       end
 
-      it "replaces values, if configured to do so" do
-        assert_equal Dummy::Schema::MAGIC_INT_COERCE_VALUE, GraphQL::Types::Int.coerce_result(99**99, context)
-      end
```

Comment:
Does this have to be removed? It seems like the override in the example schema _could_ be retained, but a method call would have to be changed.... or is this behavior not possible anymore? 

(Maybe the old override could be retained as-is if an alias accessor was added to the new IntegerEncodingError)

Reply:
No it doesn't have to be I don't think. I'll check